### DISCUSSION TITLE
Apply best practices from TYPO3-Documentation/tea

### DISCUSSION THREAD

**Discussion by nostadt** [FriendsOfTYPO3/extension_builder#363] [Ideas]
What do you think of applying best practices from that extension? Each best practice that should be applied becomes an individual issue and is either associated with a new milestone "Apply best practices" or with the existing "TYPO3 v10 compatibility".

Assuming the changes will be applied to v10 only it can make sense to associate it with the existing milestone.

Some low hanging fruits:

- normalize composer & use sort-package
- TER auto deployment
- probably linter, depending on how much the linter detect

**Comment by simonschaufi** [comment]
There is budget from the association available to implement that so yes, we already considered that idea

**Reply by nostadt** [reply]
Great
